### ISSUE TITLE
[Bug] Category import broken on Sylius 1.10.4 and PHP 8.0

### ISSUE THREAD

**Issue by plewandowski** [synolia/SyliusAkeneoPlugin#85]
**Describe the bug**
Only root category is imported. Sub categories are not imported.

it seems that it is caused by re-assigning parent and root categories after saving taxon without parent in `Synolia\SyliusAkeneoPlugin\Task\Category\CreateUpdateEntityTask`

It is also reflected in logs:
```
11:35:09 INFO      [akeneo] Category "motorsensen_freischneider" will be created
11:35:09 WARNING   [akeneo] Root cannot be changed manually, change parent instead
```

**To Reproduce**
1. Run `php bin/console akeneo:import:categories -vvv`
3. Check taxons in admin (should be missing)

**Expected behavior**
Sub-categories are imported correctly

**Additional context**
Sylius 10.0.4, PHP 8.0

**Comment by TheGrimmChester** [comment]
@plewandowski, do you still have the problem? I don't reproduce it on Sylius 1.10.8 and PHP 8.

**Comment by oallain** [comment]
Fixed on [v3.0.0](https://github.com/synolia/SyliusAkeneoPlugin/releases/tag/v3.0.0)
